myuw/templates/handlebars/card/calendars/acad_cal_sp.html

```diff
@@ -13,9 +13,9 @@ <h2 class="myuw-card-title">Academic Calendar</h2>
 	        {{ /each }}
 	    </ul>
    </div>
-    {{else}}
+	<div class="view-all"><a href="/academic_calendar/" title="Navigate to Notices">View all events</a></div>
```

Comment:
Should the title here say something like "Navigate to the Calendar"? "Notices" seems weird here, but maybe I'm missing something.

Reply:
Good catch. the Link title and the message are updated.